sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosBridgeInternal.java

```diff
@@ -94,4 +95,10 @@ public static CosmosClientBuilder cloneCosmosClientBuilder(CosmosClientBuilder b
 
         return copy;
     }
+
+    public static CosmosClientBuilder setDateKeyProvider(CosmosClientBuilder cosmosClientBuilder,
+                                                         DataEncryptionKeyProvider dataEncryptionKeyProvider) {
+        cosmosClientBuilder.dataEncryptionKeyProvider(dataEncryptionKeyProvider);
```

Comment:
Will it be revisited once the stream contracts are available?

Reply:
yes. I am committed to that.